src/main/java/net/minecraftforge/common/config/ConfigElement.java

```diff
@@ -360,4 +362,40 @@ public Object getMaxValue()
     {
         return isProperty ? prop.getMaxValue() : null;
     }
+    
+    public static ConfigElement from(String modid)
+    {
+        return from(modid, null);
+    }
+    
+    /**
+     * Provides a ConfigElement which contains
+     * @param modid
+     * @param name
+     * @return
+     */
+    public static ConfigElement from(String modid, @Nullable String name)
+    {
+        return from(modid, name, "general");//'general' is default in @Config annotation
+    }
+    
+    /**
+     * Provides a ConfigElement derived from the annotation-based config system
+     * @param modid The modid stated in the {@code @Config} annotation.
+     * @param name The name stated in the {@code @Config} annotation.
+     * @param categoryName The category name stated in the {@code @Config} annotation.
+     * @return A ConfigElement based on the described category.
+     */
+    public static ConfigElement from(String modid, @Nullable String name, @Nullable String categoryName)
```

Comment:
if all of this is stated in the @Config annotation why not just pull int he class with the @config annotation?